parcels/field.py

```diff
@@ -265,11 +258,15 @@ def from_netcdf(cls, name, dimensions, filenames, indices={},
                1. spherical (default): Lat and lon in degree, with a
                   correction for zonal velocity U near the poles.
                2. flat: No conversion, lat/lon are assumed to be in m.
+        :param full_load: boolean whether to fully load the data or only pre-load them. (default: False)
+               It is advised not to fully load the data, since in that case Parcels deals with
+               a better memory management during particle set execution.
+               full_load is however necessary for plotting the fields.
```

Comment:
Add 'sometimes' to this sentence. I.e. `full_load is however sometimes necessary for plotting the fields`